readme.md

```diff
@@ -0,0 +1,18 @@
+# TGM Example Plugin
+[![GitHub license](https://img.shields.io/badge/license-GPLv3-blue.svg)](https://raw.githubusercontent.com/TGMPA/TGM-Plugin-Activation/develop/LICENSE.md)
+[![Build Status](https://travis-ci.org/TGMPA/tgm-example-plugin.svg?branch=master)](https://travis-ci.org/TGMPA/tgm-example-plugin)
+
+## Description
+
+Example plugin to demonstrate the ability to handle bundled plugins with the TGM Plugin Activation library.
+
+Please visit the [TGM Plugin Activation](http://tgmpluginactivation.com/) website for more information.
+
+
+## Changelog
```

Comment:
"Change Log" is the correct term - the non-spaced versions is only for filenames.